### PR TITLE
Fix SSIMLoss docstring examples: remove redundant 1- prefix

### LINKED ISSUE #8822: [Bug] `SSIMLoss` docstring examples are incorrect: `1-SSIMLoss()(x,y)` negates the loss
## Bug Description

The docstring examples in `SSIMLoss.forward()` (file: `monai/losses/ssim_loss.py`, ~line 100) all show:

```python
print(1-SSIMLoss(spatial_dims=2)(x,y))
```

However, `SSIMLoss.forward()` already computes `1 - ssim_value` internally (line ~107):
```python
loss: torch.Tensor = 1 - ssim_value
```

So calling `1-SSIMLoss()(x,y)` produces `1 - (1 - ssim) = ssim`, which is the **structural similarity value** — not the loss. This confuses users into thinking they need to subtract from 1 again, leading to incorrect training.

## Expected Behavior

The docstring examples should read:
```python
print(SSIMLoss(spatial_dims=2)(x,y))
```

## Impact

Users following these examples would compute a "loss" that actually **increases** when images become less similar, which is the **opposite** of what is intended. Any training loop based on these examples would train in the wrong direction.

## Location

`monai/losses/ssim_loss.py`, `SSIMLoss.forward` docstring examples (lines 93-113 approx)

## Proposed Fix

Remove the `1-` prefix from each of the three docstring example `print()` statements:

```python
# 2D data
print(SSIMLoss(spatial_dims=2)(x,y))  # was: print(1-SSIMLoss(spatial_dims=2)(x,y))

# pseudo-3D data
print(SSIMLoss(spatial_dims=2)(x,y))  # was: print(1-SSIMLoss(spatial_dims=2)(x,y))

# 3D data
print(SSIMLoss(spatial_dims=3)(x,y))  # was: print(1-SSIMLoss(spatial_dims=3)(x,y))
```

## Chain Context

This issue was found as part of a systematic audit of `monai/losses/` for correctness:
- PR #8818: `image_dissimilarity.py` — `register_buffer` fix for `require_grads`
- Issue #8819: systematic `register_buffer` audit
- Issue #8820 / PR #8821: `spectral_loss.py` — `JukeboxLoss` variable swap fix
- **This issue**: `ssim_loss.py` — docstring example bug

### PR DESCRIPTION
**Pull request by haoyu-haoyu**
Fixes #8822.

### Description

\`SSIMLoss.forward()\` already returns \`1 - ssim_value\` (see \`monai/losses/ssim_loss.py\` line 127):

\`\`\`python
ssim_value = self.ssim_metric._compute_tensor(input, target).view(-1, 1)
loss: torch.Tensor = 1 - ssim_value
\`\`\`

…yet the three docstring examples call \`print(1-SSIMLoss(...)(x,y))\`, which computes \`1 - (1 - ssim) = ssim\` — the structural similarity value, not the loss. A user copying the example would build a "loss" that **increases** as images grow more similar and train in the wrong direction.

Removing the \`1-\` prefix in the three \`print(...)\` calls makes the examples print the actual loss, consistent with the \`Returns\` docstring that already says *"1 minus the ssim index (recall this is meant to be a loss function)"*.

Verified locally:

\`\`\`
>>> SSIMLoss(spatial_dims=2)(x, x)  # identical inputs
tensor(0.)   # correct loss behavior
\`\`\`

Before the fix the docstring would have printed \`1.0\` (the similarity), after the fix it prints \`0.0\` (the loss).

### Types of changes
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] In-line docstrings updated.

### Checks
- [x] I've read the [contribution guidelines](https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md).
- [x] DCO \`Signed-off-by\` included.

### CODE REVIEW

**Comment by ericspod** [comment]
Thanks for the update @haoyu-haoyu but this is being addressed in #8821 which will be merged soon.

**Comment by haoyu-haoyu** [comment]
Hi @ericspod — thanks for the heads-up! Just to double-check, I think #8821 might be the wrong number — that PR is @Zeesejo's fix for the variable swap in `JukeboxLoss.forward` (issue #8820), and its diff only touches `monai/losses/spectral_loss.py`, not `ssim_loss.py`.

I scanned all PRs that touch `ssim_loss.py` and only #8828 (this PR) is open. Did you mean a different PR, or were you grouping this in with the broader `monai/losses/` audit cluster (#8818, #8821, #8828) and planning to take all of them in together?

Happy to close this if there's an SSIMLoss fix landing elsewhere — just want to make sure we don't drop the docstring fix on the floor.

**Comment by ericspod** [comment]
> Hi @ericspod — thanks for the heads-up! Just to double-check, I think #8821 might be the wrong number — that PR is @Zeesejo's fix for the variable swap in `JukeboxLoss.forward` (issue #8820), and its diff only touches `monai/losses/spectral_loss.py`, not `ssim_loss.py`.
> 
> I scanned all PRs that touch `ssim_loss.py` and only #8828 (this PR) is open. Did you mean a different PR, or were you grouping this in with the broader `monai/losses/` audit cluster (#8818, #8821, #8828) and planning to take all of them in together?
> 
> Happy to close this if there's an SSIMLoss fix landing elsewhere — just want to make sure we don't drop the docstring fix on the floor.

I see #8821 changing `ssim_loss.py` in the same way as your PR, this was with a later update commit adding to the original changes in `spectral_loss.py`. Could you please double check that PR again, I see commit 4417d79b54de36f2677fc32e70cfa246a104f097 adding the  `ssim_loss.py` change.

**Comment by haoyu-haoyu** [comment]
Thanks @ericspod — you're right, I missed that commit. I looked at 4417d79b54de36f2677fc32e70cfa246a104f097 on #8821 and it does the exact same three-`print(1-SSIMLoss(...))` → `print(SSIMLoss(...))` edits as this PR. Closing this one to avoid duplication. Thanks @Zeesejo for the broader losses audit.
